hermes-common/src/main/java/pl/allegro/tech/hermes/common/http/MessageMetadataHeaders.java

```diff
@@ -3,7 +3,8 @@
 public enum MessageMetadataHeaders {
 
     MESSAGE_ID("Hermes-Message-Id"),
-    TOPIC_NAME("Hermes-Topic-Name");
+    TOPIC_NAME("Hermes-Topic-Name"),
+    TRACE_ID("Hermes-Trace-Id");
```

Comment:
As far as I know services uses 'Trace-Id' header. Since it is the same value I think we should also use same name for the header.